packages/model/src/event.ts

```diff
@@ -77,7 +77,7 @@ export interface EventResponse {
 
   calendar: CalendarResponse;
   group?: GroupResponse;
-  workingGroup?: WorkingGroupResponse;
+  workingGroup?: WorkingGroupDataObject;
```

Comment:
incorrect, you shouldn't mix these. This Response type should only reference another Response type